src/main/java/com/ace/alfox/lib/Database.java

```diff
@@ -1,48 +1,34 @@
 package com.ace.alfox.lib;
 
-import com.ace.alfox.game.interfaces.IAction;
 import org.iq80.leveldb.DB;
 import org.iq80.leveldb.Options;
 import org.springframework.beans.factory.config.BeanDefinition;
 import org.springframework.context.annotation.ClassPathScanningCandidateComponentProvider;
 import org.springframework.core.type.filter.AnnotationTypeFilter;
 import org.springframework.stereotype.Service;
 
-import static org.fusesource.leveldbjni.JniDBFactory.*;
-
 import java.io.File;
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
 
+import static org.fusesource.leveldbjni.JniDBFactory.factory;
+
+@Service
 public class Database {
-    /**
-     * Mappings of /perform/{action} to an IAction. Populated in the AlfoxApplication class.
-     */
-    public static Map<String, Class<IAction>> actions = new HashMap<>();
     private static DB instance;
 
-    private Database() {}
-
-    public static void findActions(String namespace) throws ClassNotFoundException {
-        ClassPathScanningCandidateComponentProvider scanner = new ClassPathScanningCandidateComponentProvider(true);
-        scanner.addIncludeFilter(new AnnotationTypeFilter(PlayerAction.class));
-        for (BeanDefinition bd : scanner.findCandidateComponents(namespace)) {
-            // use my annotation to get simple name
-            String alias = Class.forName(bd.getBeanClassName()).getAnnotation(PlayerAction.class).alias();
-            System.out.println(alias);
-            Class cl = Class.forName(bd.getBeanClassName());
-            Database.actions.put(alias, cl);
-        }
+    public Database() {}
+
+    public DB get() throws IOException {
+        return get("Game");
     }
 
-    public synchronized static DB get() throws IOException {
+    public synchronized DB get(String name) throws IOException {
         if(Database.instance != null)
             return Database.instance;

```

Comment:
Generally good practice to always use `{}` in Java. Really easy for someone to come along and add an indented print statement or something and break the functionality without realizing it. *horror flashbacks*